inc/page_cache/namespace.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Altis\Cloud\Page_Cache;
+
+use function Altis\Cloud\get_config;
+
+function bootstrap() {
+	global $batcache;
+	$batcache = [];
+
+	// Ignore common query params.
+	ignore_query_params();
+
+	// Forward CloudFront cache headers.
+	handle_cloudfront_headers();
+}
+
+function ignore_query_params() {
+	global $batcache;
+	$batcache = $batcache ?? [];
+
+	$config = get_config();
+	$ignored_query_string_params = $config['cache']['ignored-query-string-params'] ?? [
```

Comment:
Could this be part of the defaults in the module?

Reply:
Yeah, wasn't sure of the overall approach yet so just put them there for testing.